propolis/src/hw/nvme/mod.rs

```diff
@@ -316,13 +305,73 @@ impl PciNvme {
             ..Default::default()
         });
 
+        // We have unit tests that these are 16 and 64 bytes, respectively
+        // But just make sure as we specify these as powers of 2 in places
+        debug_assert_eq!(mem::size_of::<RawCompletion>().count_ones(), 1);
+        debug_assert_eq!(mem::size_of::<RawSubmission>().count_ones(), 1);
```

Comment:
style nit: this could assert `is_power_of_two()` instead of `count_ones()`, right?